avm/res/network/private-endpoint/main.bicep

```diff
@@ -239,7 +239,7 @@ type manualPrivateLinkServiceConnectionsType = {
   @description('Required. Properties of private link service connection.')
   properties: {
     @description('Required. The ID of a group obtained from the remote resource that this private endpoint should connect to.')
```

Comment:
I think the description should contain something like "when using private link service groupId can be defined as empty string array". What do you think @segraef?

Reply:
Agreed.

Reply:
Added.